Content.Shared/Mind/Filters/AliveHumansPool.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Shared.Mind.Filters;
+
+/// <summary>
+/// A mind pool that uses <see cref="SharedMindSystem.AddAliveHumans"/>.
+/// </summary>
+public sealed partial class AliveHumansPool : IMindPool
+{
+    void IMindPool.FindMinds(HashSet<Entity<MindComponent>> minds, EntityUid? exclude, IEntityManager entMan, SharedMindSystem mindSys)
+    {
+        mindSys.AddAliveHumans(minds, exclude);
```

Comment:
Is the choice to add `AddAliveHumans` an optimization decision, as opposed to using the existing `GetAliveHumans` method and transposing it in this method?

Reply:
yes AddAliveHumans just adds to an existing hashset instead of allocating a new one every time